README.md

```
@@ -49,11 +49,11 @@ cmake -B .build && cmake --build .build
 - [x] .OBJ files
 - [x] Limited cones 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)